virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -60,6 +52,8 @@ include::modules/virt-networking-wizard-fields-web.adoc[leveloffset=+2]
 
 include::modules/virt-attaching-vm-secondary-network-cli.adoc[leveloffset=+2]
 
+include::modules/virt-networking-glossary.adoc[leveloffset=+1]
```

Comment:
I agree that this module was not in a good place but I am not sure anyone will find it here. Personally, I find that glossary sections get ignored. Also, some of the terms could be removed from the glossary, although that is out of scope for your task. PXE, CRD, and NAD are not specific to CNV, so I am not sure they they were every included.

Reply:
Let's ask about this. I think there is a better way to present this info than in Glossary format, which by it's nature tends to get skimmed or not read at all (because we're not putting it in any relatable context).